sporkify/frontend/views.py

```diff
@@ -95,6 +95,18 @@ def inventory(request):
     })
 
 
+@login_required
+def delete_inventory(request):
+    if request.method == 'POST':
+        form = Inventory()
+        inventory = Inventory.objects.all()
+        item_id = request.POST.get('product_code')
+        item = Inventory.objects.get(product_code=item_id)
+        item.delete()
+        return render(request, 'inventory.html', {
```

Comment:
If you pass your inv objects in the context here, they'll show after deleting an item.

Reply:
#JoshDjangoMasterFerrara